CAIPs/caip-28.md

```diff
@@ -0,0 +1,62 @@
+---
+caip: 28
+title: Blockchain Reference for Stellar Namespace
+author: Gleb Pitsevich (@pitsevich)
+discussions-to: TBD
+status: Draft
+type: Standard
+created: 2021-02-17
+requires: 2
+---
+
+## Simple Summary
+
+This document is about the details of the Stellar network namespace and reference for CAIP-2.
+
+## Abstract
+In CAIP-2 a general blockchain identification scheme is defined. This is the
+implementation of CAIP-2 for the Stellar network.
+
+## Motivation
+See CAIP-2.
+
+## Specification
+
+### Filecoin Namespace
+
+The namespace "stellar" refers to the wider Stellar ecosystem.
+
+#### Reference Definition
+
+The reference relies on Stellar's current designation of addresses belonging to test or main networks by prefixing them
+with `testnet` or `pubnet` correspondingly.
+
+Reference could only be populated with `testnet` or `pubnet` symbols.
+
```

Comment:
Missing "Resolution Method" section which I'm assuming was also missing on the Filecoin spec 

Refer to [EIP155 namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-3.md) for an example using JSON-RPC and [Cosmos namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-5.md) for an example using REST

Reply:
@pedrouid Thanks for the feedback, added this section.